extensions/publora/src/api.tsx

```diff
@@ -0,0 +1,37 @@
+import { Platform, Post } from "./types";
+
+export const fetchPlatforms = async (api_key: string): Promise<Platform[]> => {
+  const response = await fetch("https://api.publora.com/api/v1/platform-connections", {
+    headers: {
+      "x-publora-key": `${api_key}`,
+    },
+  });
+
+  if (!response.ok) {
+    const errorData = await response.json();
+    throw new Error(`Error fetching platforms: ${errorData.message}`);
+  }
+
+  const data = await response.json();
+
+  return data.connections as Platform[];
+};
+
+export const schedulePost = async (api_key: string, post: Post) => {
+  const body = JSON.stringify(post);
+  const response = await fetch("https://api.publora.com/api/v1/create-post", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "x-publora-key": `${api_key}`,
+    },
+    body: body,
+  });
+
+  if (!response.ok) {
+    const errorData = await response.json();
+    throw new Error(`Error scheduling post: ${errorData.message}`);
+  }
+
+  return await response.json();
```

Comment:
Be careful when writing `return await`. See here: https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise.

```suggestion
  return response.json();
```

Reply:
Thank you 🙏 I'm mostly Elixir dev for now. I have no too much experience in JS / TS. I'll educate myself.

Reply:
Fixed